assets/css/styles.css

```diff
@@ -0,0 +1,250 @@
+:root {
+  /* CSS HEX */
+  --jet: #353535ff;
+  --ming: #3c6e71ff;
+  --white: #ffffffff;
+  --gainsboro: #d9d9d9ff;
+  --indigo-dye: #284b63ff;
+  --lightslategrey: #778899;
+}
+/* universal element */
+* {
+  box-sizing: border-box;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+body {
+  /* background-color: var(--white); */
+  background-image: url(./assets/images/backgroundimage.jpeg);
+  line-height: 1.2;
+}
+
+/* navbar */
+
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  align-content: flex-start;
+  border: none;
+  border-radius: 10px;
+  box-shadow: var(--gainsboro);
+  background-color: var(--jet);
+  margin-bottom: 5vh;
+}
+.nav-item {
+  display: flex;
+  flex-direction: row;
+  border: solid 2px var(--gainsboro);
+  margin: 10px;
+  border-color: var(--jet);
+}
+
+/* pseudo element for the nav buttons */
+.nav-item:hover {
+  border-color: var(--white);
+  color-adjust: text;
+  font-variant: var(--jet);
+}
+.nav-titles {
+  text-decoration: none;
+  color: var(--gainsboro);
+}
+
+/* about-me */
+.about-me-container {
+  display: flex;
+  flex-direction: row;
+  border: none;
+  margin: 2%;
+  color: var(--white);
+  background-color: var(--ming);
+  margin-bottom: 10vh;
+  box-shadow: rgba(240, 46, 170, 0.4) 5px 5px, rgba(240, 46, 170, 0.3) 10px 10px,
+    rgba(240, 46, 170, 0.2) 15px 15px, rgba(240, 46, 170, 0.1) 20px 20px,
+    rgba(240, 46, 170, 0.05) 25px 25px;
+}
+
+/* about me contains image icon and text */
+
+.about-me-item {
+  display: column;
+  flex-direction: row;
+  position: relative;
+  margin: 0%;
+}
+
+.profile-image {
+  max-height: 100%;
+  max-width: 100%;
+  padding: 5px;
+  border-radius: 25px;
+  object-fit: contain;
+}
+
+.wave {
+  display: inline-block;
+}
+
+/* css for semantic html refactor project */
+
+/* header tags on page */
+.hr-lines {
+  text-align: center;
+}
+
+/* first project- Horiseon Marketing agency */
+.mainproject {
+  background-color: var(--lightslategrey);
+  margin-bottom: 5vh;
+  width: 100vw;
+}
+
+.projectmain {
+  width: 100vw;
+  height: 60vh;
+  object-fit: contain;
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px,
+    rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px,
+    rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+  border-image-width: 10px;
+}
+
+h3 {
+  text-align: center;
+}
+
+/* other projects */
+.project-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  height: 40v;
+  align-items: center;
+  background-color: var(--indigo-dye);
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px,
+    rgba(0, 0, 0, 0.3) 0px 30px 60px -30px,
+    rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;
+  margin-bottom: 10vh;
+}
+
+.project-items {
+  display: flex;
+  flex-direction: row;
+  border-left: 5px solid black;
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+  height: 30rem;
+  width: 40vw;
```

Comment:
A fixed width on the projects causes them to look very narrow on a mobile viewport. Add the width to media queries to fix this.